app/assets/javascripts/initializePage.js

```diff
@@ -46,7 +46,6 @@ function callInitializers() {
   initializeArticleDate();
   initializeArticleReactions();
   initNotifications();
-  initializeStylesheetAppend();
```

Comment:
I think we want to remove the reference to this (even though it's just a comment) in line 2 of this file

Reply:
Good call.